### PR TITLE
Fixes some mods.toml issues

### PR DESCRIPTION
**Pull request by ravenbuilder934**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- Makes URL (minecolonies.com) and logo show up in the mods menu
- Bumps required forge version to match required MC version
- Fixes issue link (old one didn't point to issue templates)
- Fixes description up a bit

Review please

### CODE REVIEW

File: src/main/resources/META-INF/mods.toml [hunk 1]
```diff
@@ -23,23 +23,23 @@ displayName="Structurize" #mandatory
...
+An astonishing way to copy & paste structures!
 
 Allows you to:
  - save structures of your choice into a file of small size
- - load them back and place anywhere anytime
- - also upload them and share between others
- - create various shapes using math formula
- - and many other nice things a great builder would love to have
+ - load them back and place them anywhere anytime
+ - upload them and share with others
```
Comment (Nightenom) [inline src/main/resources/META-INF/mods.toml]: lol, it's still here :D old copypasta from new branch
```suggestion
```

File: src/main/resources/META-INF/mods.toml [hunk 2]
```diff
@@ -48,7 +48,7 @@ Allows you to:
     # Does this dependency have to exist - if not, ordering below must be specified
     mandatory=true #mandatory
     # The version range of the dependency
-    versionRange="[32.0.0,)" #mandatory
+    versionRange="[34.0.0,)" #mandatory
```
Comment (Nightenom) [inline src/main/resources/META-INF/mods.toml]:
```suggestion
    versionRange="[34.1.0,)" #mandatory
```

**Review by Nightenom — COMMENTED** [review]
Does the logo work or crash?

**Comment by ravenbuilder934** [comment]
One sec, I'll test the logo

**Comment by ravenbuilder934** [comment]
It doesn't crash, but the logo doesn't show up either. Is [this](https://github.com/ldtteam/Structurize/blob/version/1.16.3/src/main/resources/assets/structurize/structurize.png) the wrong place for the logo?

**Comment by Nightenom** [comment]
> It doesn't crash, but the logo doesn't show up either. Is [this](https://github.com/ldtteam/Structurize/blob/version/1.16.3/src/main/resources/assets/structurize/structurize.png) the wrong place for the logo?

Don't know, search yourself :P

**Comment by ravenbuilder934** [comment]
*checks mcol's logo* Ah, nope, needs to be back 2 levels. give me a min

**Comment by ravenbuilder934** [comment]
Done

**Comment by Raycoms** [comment]
:shipit:

**Pull request merged**